src/pages/PlanPage/subRoutes/CurrentOrgPlan/queries/EnterpriseAccountDetailsQueryOpts.ts

```diff
@@ -15,7 +15,7 @@ const AccountSchema = z.object({
 
 export type Account = z.infer<typeof AccountSchema>
 
-const RequestSchema = z.object({
+export const RequestSchema = z.object({
```

Comment:
IMO rename this here instead of importing with an alias. We already have four other `RequestSchema`s exported lol

Reply:
Updated